test/e2e/config/config.go

```diff
@@ -4,12 +4,16 @@ import "github.com/kelseyhightower/envconfig"
 
 // Config holds global test configuration translated from environment variables
 type Config struct {
-	SubscriptionID        string `envconfig:"SUBSCRIPTION_ID"`
-	ResourceGroup         string `envconfig:"RESOURCE_GROUP"`
-	AzureClientID         string `envconfig:"AZURE_CLIENT_ID"`
-	KeyvaultName          string `envconfig:"KEYVAULT_NAME"`
-	KeyvaultSecretName    string `envconfig:"KEYVAULT_SECRET_NAME"`
-	KeyvaultSecretVersion string `envconfig:"KEYVAULT_SECRET_VERSION"`
+	SubscriptionID           string `envconfig:"SUBSCRIPTION_ID"`
+	ResourceGroup            string `envconfig:"RESOURCE_GROUP"`
+	AzureClientID            string `envconfig:"AZURE_CLIENT_ID"`
+	KeyvaultName             string `envconfig:"KEYVAULT_NAME"`
+	KeyvaultSecretName       string `envconfig:"KEYVAULT_SECRET_NAME"`
+	KeyvaultSecretVersion    string `envconfig:"KEYVAULT_SECRET_VERSION"`
+	MICVersion               string `envconfig:"MIC_VERSION"`
+	NMIVersion               string `envconfig:"NMI_VERSION"`
```

Comment:
Can you please add some notes on the README on steps to run custom images.

Reply:
done